plutus-pab/src/Plutus/PAB/Monitoring/PABLogMsg.hs

```diff
@@ -74,10 +77,10 @@ instance Pretty (ContractDef t) => Pretty (AppMsg t) where
         ContractInstances t s            -> pretty t <+> pretty s
         ContractHistoryItem instanceId s -> pretty instanceId <> colon <+> pretty (fmap encodeToLazyText s)
 
-data PABLogMsg =
+data PABLogMsg t =
```

Comment:
Why did you add this parameter?

Reply:
`t` is used throughout the PAB as the type of contracts that are run by the PAB. It's usually either `ContractExe` (for contracts that exist as executables somewhere in the file system) or `Builtin` (for contracts that are compiled with the PAB and run in the same process)

Previously this was specialised to `ContractExe` in `PABLogMsg` but I needed to use it with other types as well.